tests/helix/send-to-helix-basictests.targets

```diff
@@ -21,10 +24,14 @@
 
     <ItemGroup>
       <_DefaultWorkItems Include="$(WorkItemArchiveWildCard)" />
+      <!-- runsettings timeout in ms, default to 10 mins -->
+      <_DefaultWorkItems TimeoutMs="600000" />
+
+      <_DefaultWorkItems Condition="'%(FileName)' == 'Aspire.Hosting.Elasticsearch.Tests'" TimeoutMs="1200000" />
```

Comment:
This can't be done with a runsettings file? If this is done centrally here this will be missed out.

Reply:
It can, but you would need a copy of the `.runsettings` file for any project that is overriding it. I have some changes planned that would give the projects more control over their own helix run. But this should get us closer to green CI for now.

Reply:
Ok, as long as you intend to de-centralize it later. Something like <HelixTimeout> in projects would be great.